src/hazelweb/gui/CursorInspector.re

```diff
@@ -92,25 +96,37 @@ let view = (model: Model.t): Vdom.Node.t => {
     got_indicator("Got", special_msg_bar("a case rule"));
   let got_keyword_indicator =
     got_indicator("Got a reserved keyword", typebar(HTyp.Hole));
+  let got_duplicate_indicator =
+    got_indicator("Got a duplicated variable", typebar(HTyp.Hole));
+  let got_var_indicator = (shadow, num_of_uses) =>
+    got_indicator(
+      "Got a variable",
+      special_msg_bar(
+        (shadow ? " shadowing previous variable, " : "")
+        ++ "used "
+        ++ Int.to_string(num_of_uses)
+        ++ " times",
```

Comment:
You need to check whether the num_of_uses is 0 and write "time" instead of "times" for this to be correct english

Reply:
also the sidebar shouldn't be green when there is a duplicate variable -- maybe change the background color of the part that says "used 0 times" to be the same color as the crit unused variable outline

Reply:
![Screenshot_2020-05-05_20-47-55](https://user-images.githubusercontent.com/22940214/81129246-c1641880-8f11-11ea-9beb-70c07661390d.png)
Does this fit what in mind? Would the purple be way too highlighting?